### DISCUSSION TITLE
Add custom tag to uploaded/"consumed" documents, similar to emails from IMAP

### DISCUSSION THREAD

**Discussion by tvhdev** [jonaswinkler/paperless-ng#973] [Feature Requests]
It would be useful to add a custom tag to new uploaded/"consumed" documents, similar to the imported emails from IMAP. This way one could keep track of what documents were tagged completly and which documents were still to be completely tagged within the "saved views"

**Comment by shamoon** [comment]
This sounds like designating an “inbox” tag? (Which already exists)

**Reply by tvhdev** [reply]
Oh, you're right, i didnt see it...
